Tests/W3CTestSuite/PassingTests.txt

```diff
@@ -228,6 +228,8 @@ __issue-227-02.svg
 __issue-239-01.svg
 __issue-242-01.svg
 __issue-279-01.svg
+__issue-280-01.svg
```

Comment:
I already had noticed that the reference images have been incorrect - thanks for correcting them!